app/Http/Controllers/DemandController.php

```diff
@@ -196,34 +196,8 @@ public function click(Request $request, $id)
         //     ]);
         // }
 
-        $response = new \Symfony\Component\HttpFoundation\Response($url);
+      return  new RedirectResponse($url);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the code after this "return". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1763)